superset-frontend/plugins/plugin-chart-echarts/src/Radar/controlPanel.tsx

```diff
@@ -210,6 +210,11 @@ const config: ControlPanelConfig = {
       ],
     },
   ],
+  denormalizeFormData: formData => ({
```

Comment:
It looks like the denormalizeFormData is consistent for most charts, so maybe we can abstract out a generic function?

Reply:
Different viz have different control names, for example, the `BigNumber` supported 1 metric and 0 dimension, the metrics in SFD should pick only one, and drop other columns.